docs/en/latest/plugins/openid-connect.md

```diff
@@ -62,7 +62,7 @@ description: OpenID Connect allows the client to obtain user information from th
 | session                              | object   | False    |                       |              | When bearer_only is set to false, openid-connect will use Authorization Code flow to authenticate on the IDP, so you need to set the session-related configuration.                                                                   |
 | session.secret                       | string   | True     | Automatic generation  | 16 or more characters | The key used for session encrypt and HMAC operation.                                                                                                                                                                                  |
 | session.cookie                       | object   | False    |                       |             |                                                                                                                                                                                  |
-| session.cookie.lifetime              | integer   | False    | 3600                  |             | it holds the cookie lifetime in seconds in the future. By default this is set to 3,600 seconds. This can be configured with Nginx set $session_cookie_lifetime 3600;. This does not set cookie's expiration time on session only (by default) cookies, but it is used if the cookies are configured persistent with session.cookie.persistent == true. |
+| session.cookie.lifetime              | integer   | False    | 3600                  |             | it holds the cookie lifetime in seconds in the future. By default this is set to 3,600 seconds. |
```

Comment:
```suggestion
| session.cookie.lifetime              | integer   | False    | 3600                  |             | Cookie lifetime in seconds. |
```

Reply:
No need to specify again the default lifetime in second, as it's already specified in the other column.